modules/tensor_mechanics/include/materials/RecomputeRadialReturnPowerLawCreep.h

```diff
@@ -0,0 +1,66 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#ifndef RECOMPUTERADIALRETURNPOWERLAWCREEP_H
+#define RECOMPUTERADIALRETURNPOWERLAWCREEP_H
+
+#include "RecomputeRadialReturn.h"
+#include "MooseMesh.h"
+
+/**
+ * This class uses the Discrete material in a radial return isotropic plasticity
+ * model.  This class is one of the basic
+ * radial return constitutive models; more complex constitutive models combine
+ * creep and plasticity.
+ *
+ * This class inherits from RecomputeRadialReturnStressIncrement and must be used
+ * in conjunction with ComputeRadialReturnMappingStress.  This class calculates
+ * an effective trial stress, an effective scalar plastic strain
+ * increment, and the derivative of the scalar effective plastic strain increment;
+ * these values are passed to the RecomputeRadialReturnStressIncrement to compute
+ * the radial return stress increment.  This isotropic plasticity class also
+ * computes the plastic strain as a stateful material property.
+ *
+ * This class is based on the implicit integration algorithm in F. Dunne and N.
+ * Petrinic's Introduction to Computational Plasticity (2004) Oxford University
+ * Press, pg. 146 - 149.
```

Comment:
Please update the comments to reflect the creep model from the isotropic plasticity model.  The generic viscoplasticity algorithm is given on pg 161-163 of the Dunne and Petrinic book.  Thank you!